src/App.js

```diff
@@ -1,10 +1,31 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, {useState} from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
+
+
 function App() {
-  //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+  //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  
+  // You'll need one for the home score and another for the away score.
+  const [scoreH, setScoreH] = useState(0);
+  const [scoreA, setScoreA] = useState(0);
```

Comment:
Try and use better names, like setScoreAway, scoreAway, and also the same for Home